victims/data.py

```diff
@@ -43,10 +43,17 @@ def serialize_cases(self, buffer):
             data = {new: case.get(old) for old, new in CASE_LABELS}
 
             data["stories"] = data.get("stories") or []  # avoids getting None
+
+            # serializing tags
             data["tags"] = data.get("tags") or ""  # avoids getting None
             data["tags"] = data["tags"].split("|")
             data["tags"] = [tag.strip().lower() for tag in data["tags"] if tag]
 
+            # serializing  latitude and longitude
+            data["latlng"] = data["latlng"].strip() or ""
+            data["latlng"] = data["latlng"].split(",")
+            data["latlng"] = list(map(float, data["latlng"]))
```

Comment:
I think this is slightly more readable, what do you think?

```suggestion
            data["latlng"] = [float(coord) for coord in data["latlng"]]
```

Reply:
BTW if this column is empty this line breaks the app… we might need something for this edge case ; )

Reply:
Yes! I've started by adding some try/excptions for `ValueError` exceptions but i am not sure what you guys prefer.